plugins/karate/Makefile.mk

```diff
@@ -1,8 +1,14 @@
 # LIBRARIES
 ##################################################
 if USE_KARATE
+# Plugin description is generated from README.md
+BUILT_SOURCES += plugins/karate/PluginDescription.h
+plugins/karate/PluginDescription.h: plugins/karate/README.md plugins/karate/Makefile.mk plugins/convert_README_to_header.sh
+	sh $(top_srcdir)/plugins/convert_README_to_header.sh plugins/karate PluginDescription.h
```

Comment:
Ah it looks like it's currently being built so early it doesn't break the distcheck, but I'm not sure how. Anyway I think it needs changing please.